Makefile

```diff
@@ -145,7 +145,7 @@ INSTALL_FILES = $(INSTALL) -m 644
 INSTALL_PLUGIN_DIR = $(DESTDIR)$(ABS_PLUGIN_DIR)
 
 install: $(INSTALL_PLUGIN_DIR)
-	$(INSTALL_FILES) $(RELEASE_LIB) $(INSTALL_PLUGIN_DIR)
+	$(INSTALL) -m 755 $(RELEASE_LIB) $(INSTALL_PLUGIN_DIR)
```

Comment:
I think that was the only use of INSTALL_FILES macro, it can be removed then.

Reply:
I dropped it in the revised commit. Please review